pinot-core/src/main/java/org/apache/pinot/core/transport/InstanceRequestHandler.java

```diff
@@ -54,16 +54,17 @@
 public class InstanceRequestHandler extends SimpleChannelInboundHandler<ByteBuf> {
   private static final Logger LOGGER = LoggerFactory.getLogger(InstanceRequestHandler.class);
 
-  // TODO: make it configurable
-  private static final int SLOW_QUERY_LATENCY_THRESHOLD_MS = 100;
+  private final long _slowQueryLatencyThresholdMs;
```

Comment:
This should be set directly from config instead of setting it in constructor.

Reply:
Updated to use a static field and set the variable in `ServerInstance`.